pkg/storage/memstore/memstore.go

```diff
@@ -129,3 +129,12 @@ func (m *MemStore) close() {
 
 	m.db = make(map[string][]byte)
 }
+
+// Delete deletes the key-value pair associated with k.
+func (m *MemStore) Delete(k string) error {
```

Comment:
should return an error if `k` is not found

Reply:
Done